### PR TITLE
format stackframe name with ClassName.MethodName(ParameterTypes)

### PR DESCRIPTION
**Pull request by testforstephen**
Signed-off-by: Jinbo Wang <jinbwan@microsoft.com>

add for feature https://github.com/Microsoft/java-debug/issues/43

### CODE REVIEW

**Comment by testforstephen** [comment]
image

File: com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/StackTraceRequestHandler.java [hunk 1]
```diff
@@ -152,4 +154,21 @@ public void handle(Command command, Arguments arguments, Response response, IDeb
             }
         }
     }
+
+    private String formatMethodName(Method method, boolean showContextClass, boolean showParameter) {
+        String formattedName = method.name();
```
Comment (yaohaizh) [inline com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/StackTraceRequestHandler.java]: Better to use StringBuilder in this method for lots of String concat

Comment (andxu) [reply com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/StackTraceRequestHandler.java]: why not use typeformatter

Comment (testforstephen) [reply com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/StackTraceRequestHandler.java]: fixed.

**Pull request merged**